app/dashboard/templates/shared/zero_balance_error.html

```diff
@@ -15,25 +15,28 @@
   along with this program. If not, see <http://www.gnu.org/licenses/>.
 {% endcomment %}
 {% load i18n static %}
-<div id="zero_balance_error" class="blocking_error" style="display: none; min-height: 500px;">
-  {% include 'shared/bounty_nav.html' with active_item=1 %}
-  <img src="{% static "v2/images/prime.png" %}">
-  <br>
-  <h3>{% trans "You're almost there!" %}</h3>
-  <p>
-    {% trans "In order to continue, you'll need some Ether." %}
-  </p>
-  <p>
-    {% trans "'Ether' is a digital asset that can be used to pay for the computational resources needed to run an application or program. In this case, Gitcoin." %}
-  </p>
-  <p>
-    {% trans "The easiest way to get Ether is to request it via the Gitcoin Faucet:" %}
-  </p>
-  <p class="float-right">
-    <a class="btn btn-sm btn-info mt-3 pulseClick" role="button" id="submit" href="/faucet" target="_blank" rel="noopener noreferrer">{% trans "View Faucet" %}</a>
-    <br>
-    - {% trans "or" %} -
-    <br>
-    <a href="https://github.com/gitcoinco/gitcoinco/issues/4" target="_blank" rel="noopener noreferrer">{% trans "Can't get web3 to work?  Get help here." %}</a>
-  </p>
+<div id="zero_balance_error" class="submit-notification metamask-error container-fluid" style="display: none;">
+  <div class="row align-items-center px-4 pt-2 text-center text-lg-left">
+    <div class="col-lg-10 col-sm-12">
+      <h3>{% trans "You're almost there!" %}</h3>
+      <p>
+        {% trans "In order to continue, you'll need some Ether." %}
+      </p>
+      <p>
+        {% trans "'Ether' is a digital asset that can be used to pay for the computational resources needed to run an application or program. In this case, Gitcoin." %}
+      </p>
+      <p>
+        {% trans "The easiest way to get Ether is to request it via the Gitcoin Faucet:" %}
+      </p>
+    </div>
+    <div class="col-lg-2 col-sm-12">
+      <p>
+        <a class="btn btn-sm btn-info mt-3 pulseClick" role="button" id="submit" href="/faucet" target="_blank" rel="noopener noreferrer">{% trans "View Faucet" %}</a>
```

Comment:
This is merely a note since it was already like this, but we'll want to be mindful of replacing hard coded URLs in templates with `{% url 'url_reverse_name' %}`